holo-ospf/src/ospfv2/packet/lsa_opaque.rs

```diff
@@ -122,6 +122,8 @@ pub struct LsaRouterInfo {
     pub srlb: Vec<SrLocalBlockTlv>,
     pub msds: Option<MsdTlv>,
     pub srms_pref: Option<SrmsPrefTlv>,
+    // #[serde(skip)]
```

Comment:
we need `#[serde(skip)]` here otherwise some conformance tests will fail. This is a workaround for something that should be fixed in the test framework soonish.